src/custom/pages/Swap/index.tsx

```diff
@@ -375,6 +386,132 @@ const SwapButton = ({ children, showLoading, showButton = false }: SwapButtonPro
     </Text>
   )
 
+interface HighFeeContainerProps {
+  padding?: string
+  margin?: string
+  width?: string
+}
+
+// const ShowMoreButton = styled.span`
+//   text-decoration: underline;
+//   cursor: pointer;
+//   margin-left: auto;
+// `
+
+const WarningCheckboxContainer = styled.small`
+  &&&& {
+    display: flex;
+    font-weight: bold;
+    gap: 2px;
+    justify-content: center;
+    align-items: center;
+  }
+`
+
+const HighFeeWarningContainer = styled(AuxInformationContainer).attrs((props) => ({
+  ...props,
+  hideInput: true,
+}))<HighFeeContainerProps>`
+  padding: ${({ padding = '5px 16px' }) => padding};
+  font-size: small;
```

Comment:
Use specific PX's, or a pre-defined font-size from a theme (we don't have that yet, but will do it at some point to normalize these).

Reply:
removed

Reply:
removed